app/src/middleware/mailApiTracker.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @module StatsTracker
+ *
+ * Log statistics for CHES Requests.
+ *
+ * @see morgan
+ *
+ * @exports StatsTracker
+ */
+const moment = require('moment');
+const morgan = require('morgan');
+
+const stackpole = require('../components/stackpole');
+
+// read statsUrls from config?
+const mailUrl = '/api/v1/email';
+const mergeUrl = '/api/v1/emailMerge';
+const statsUrls = [mailUrl, mergeUrl];
+
+const mailApiTrackerFormat = ':azp :op :txId :msgId :ts :response-time';
+
+const mailApiTracker = async (req, res, next) => {
+
+  if (statsUrls.includes(req.url)) {
+    const defaultEnd = res.end;
+    const chunks = [];
+    req._timestamp = moment.utc().valueOf();
+    req._operation = req.url === mailUrl ? 'MAIL' : 'MERGE';
+    res._transactionId = '-';
+    res._messageIds = '';
+    res._to = '';
+
+    res.end = (...restArgs) => {
+      try {
+        if (restArgs[0]) {
+          chunks.push(Buffer.from(restArgs[0]));
+        }
+        const body = Buffer.concat(chunks).toString('utf8');
+        const obj = JSON.parse(body);
+        res._transactionId = obj.txId;
+        res._messageIds = obj.messages.map(m => m.msgId).join(',');
+        // eslint-disable-next-line no-empty
+      } catch (err) {
+
```

Comment:
It's worth logging errors like this in the unlikely event they occur (i.e. esoteric JSON parse failures).
Shouldn't have no-line eslint overrides unless there's a very good reason to do so.
```log.error('mailApiTracker', err);``` should suffice.

Reply:
done